app/pages/validator/[address].vue

```diff
@@ -1,15 +1,20 @@
 <script setup lang="ts">
-const { getValidatorByAddress } = useApiStore()
-
 const route = useRoute()
-const validator = computed(() => getValidatorByAddress(route.params.address as string))
+const { data: validator } = await useFetch(`/api/v1/validators/${route.params.address}`)
```

Comment:
<img width="1149" alt="image" src="https://github.com/user-attachments/assets/737ccea5-88c4-4117-9523-bde17daa8456" />

```
:3000/api/v1/validators/NQ29%20FBVT%20B4GM%20S27H%20UBP4%201MTC%20GNKQ%20VPBT%20099M:1 
Failed to load resource: the server responded with a status of 400 (Invalid address format)
```

Now, even if clicking in a validator from the home screen you get this error